mail/sylpheed/Portfile

```diff
@@ -3,12 +3,12 @@
 PortSystem          1.0
 
 name                sylpheed
-version             3.5.1
+version             3.7.0
 revision            1
```

Comment:
when updating the `version` you should reset the `revision` to zero

Reply:
Fixed